crates/nu-cli/src/shell/filesystem_shell.rs

```diff
@@ -421,9 +425,13 @@ impl Shell for FilesystemShell {
                     dir.tag(),
                 ));
             }
+            if verbose {
+                let val = format!("mkdir: created directory {:?}", dir.item).into();
```

Comment:
A couple of thoughts here.

First, "verbose" isn't super clear about what it intends to do. Perhaps it would make more sense to name the flag `--show-created-paths`?

Second, it's hard to process this data. What if I wanted to use it in a pipeline? Given the above idea of a clearer flag, what if we instead just emitted `dir.item`?

What do you think?

Reply:
Thanks for your comments. This verbose option imitated from traditional mkdir command.
But I think your idea sounds good. I'll try to modify it to fit your idea.

Reply:
@thegedge I implemented it!